### PR TITLE
Support isolate_managed_irq in the realtime profile.

### PR DESCRIPTION
**Pull request by jmencak**
This PR supports the new isolate_managed_irq functionality in the upstream realtime profile and prepares ground for tuned-2.11.0-9.el7.noarch

### CODE REVIEW

**Comment by jmencak** [comment]
Merge after https://github.com/openshift/cluster-node-tuning-operator/pull/123
/hold

File: examples/realtime-mc.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ spec:
       [variables]
       # isolated_cores take a list of ranges; e.g. isolated_cores=2,4-7
       isolated_cores=1
+      #isolate_managed_irq=Y
```
Comment (yanirq) [inline examples/realtime-mc.yaml:15]: so this is an optional override we can use when needed ?  (to override realtime profile)

Comment (jmencak) [reply examples/realtime-mc.yaml:15]: yes, optional and can be used when needed

**Pull request merged**